visualize_packs.gemspec

```diff
@@ -18,20 +18,18 @@ Gem::Specification.new do |spec|
           'public gem pushes.'
   end
 
+  spec.executables << 'visualize_packs'
+
   # Specify which files should be added to the gem when it is released.
-  spec.files = Dir['README.md', 'lib/**/*']
+  spec.files = Dir['README.md', 'lib/**/*', "bin/**/*"]
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.6'
 
-  spec.add_dependency 'sorbet-runtime'
+  spec.add_development_dependency 'bundler'
+  spec.add_development_dependency 'sorbet-runtime'
+  spec.add_development_dependency "rspec", "~> 3.12"
+
   spec.add_dependency 'packs-specification'
-  spec.add_dependency 'parse_packwerk'
-  spec.add_dependency 'code_ownership'
-  spec.add_dependency 'rake'
-  spec.add_dependency 'ruby-graphviz'
```

Comment:
This is a happy coincidence. I had been doing some memory profiling of our app, and came across some high memory use tracking back to [graphviz re-implementing `symbolize_keys`](https://github.com/glejeune/Ruby-Graphviz/blob/c3be805fc5e62ea1326eb9bb3708a807d1178525/lib/graphviz/core_ext.rb#L28-L33), which was used instead of the one that came with ActiveSupport😱